gammapy/spectrum/tests/test_spectrum_fit.py

```diff
@@ -10,33 +10,63 @@
 from ...spectrum.spectrum_fit import SpectrumFit
 from ...utils.testing import requires_dependency, requires_data, SHERPA_LT_4_8
 from astropy.utils.compat import NUMPY_LT_1_9
-
+import logging
 
 @pytest.mark.skipif('NUMPY_LT_1_9')
 @pytest.mark.skipif('SHERPA_LT_4_8')
 @requires_dependency('sherpa')
 @requires_data('gammapy-extra')
 def test_spectral_fit():
+
+    logging.basicConfig(level=logging.INFO)
```

Comment:
@cdeil I found this usefull for  debugging test, does anything speal against using loggers within tests?